### PR TITLE
fix: mobile hide activity icon when no current wallet

### PR DESCRIPTION
**Pull request by zhouhanseng**
close https://github.com/DimensionDev/Maskbook/issues/1702

### CODE REVIEW

**Comment by septs** [comment]
why bottom border not can show?

**Comment by zhouhanseng** [comment]
> why bottom border not can show?

Not included by this commit, I will open another PR to fix this.

**Comment by zhouhanseng** [comment]
image

File: src/extension/options-page/DashboardRouters/Wallets.tsx [hunk 1]
```diff
@@ -85,6 +85,31 @@ export default function DashboardWalletsRouter() {
...
+        </IconButton>,
+        <IconButton
+            onClick={() => {
+                if (currentWallet) openAddToken({ wallet: currentWallet })
+                else openWalletCreate()
+            }}>
+            <AddIcon />
+        </IconButton>,
+    ]
+    !currentWallet && rightIcons.shift()
```
Comment (jk234ert) [inline src/extension/options-page/DashboardRouters/Wallets.tsx:112]: How about simplifying the logic like:
```
let rightIcons = [
<IconButton
            onClick={() => {
                if (currentWallet) openAddToken({ wallet: currentWallet })
                else openWalletCreate()
            }}>
            <AddIcon />
        </IconButton>,
]
if (currentWallet) {
    rightIcons.insert(THE_RESTORE_ICON, at: 0)
}
```
`rightIcons.shift` is bug-prone and confusing for others to debug & maintain

Comment (zhouhanseng) [reply src/extension/options-page/DashboardRouters/Wallets.tsx:112]: okay

**Review by septs — APPROVED** [review]

**Pull request merged**